hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -528,12 +538,11 @@ public void deleteVolume(String volume) throws IOException {
 
     ListVolumeResponse resp =
         handleError(submitRequest(omRequest)).getListVolumeResponse();
-
-
-
-    return resp.getVolumeInfoList().stream()
-        .map(item -> OmVolumeArgs.getFromProtobuf(item))
-        .collect(Collectors.toList());
+    List<OmVolumeArgs> list = new ArrayList<>(resp.getVolumeInfoList().size());
```

Comment:
The change is the same as the original lambada based implementation. Any reason for that?

Reply:
now OmVolumeArgs#getFromProtobuf throws an exception which warranted this change.